recipes/eeg_positions/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "eeg_positions" %}
+{% set version = "2.1.1" %}
+{% set python_min = "3.10" %}
```

Comment:
In the upstream `pyproject.toml` on pypi, it looks like `python_requires` is `>=3.9`. Is there a reason to make that 3.10 here? If it should be 3.9, then you can drop the `{% set python_min = "3.10" %}` since it will pull the default `python_min` of 3.9 from the global pinnings.

Reply:
Thanks for the note, 3.10 is not required, that must have been a typo of mine. I changed it.